### PR TITLE
Refactor/minor changes on Fractions

### PR DESCRIPTION
**Pull request by sandordargo**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by sandordargo** [comment]
Signed CLA. Let's wait a day.

**Comment by sandordargo** [comment]
How can I ask for a rerun of the CLA check plus add some labels?

File: Lib/fractions.py [hunk 1]
```diff
@@ -266,10 +266,7 @@ def limit_denominator(self, max_denominator=1000000):
         k = (max_denominator-q0)//q1
         bound1 = Fraction(p0+k*p1, q0+k*q1)
         bound2 = Fraction(p1, q1)
-        if abs(bound2 - self) <= abs(bound1-self):
-            return bound2
-        else:
-            return bound1
+        return bound2 if abs(bound2 - self) <= abs(bound1-self) else bound1
```
Comment (vstinner) [inline Lib/fractions.py:269]: I prefer to write the code on multiple lines, IMHO it's more readable.

**Review by vstinner — CHANGES_REQUESTED** [review]
This PR is mostly made on coding style changes, and usually we don't accept PR only changing the coding style. Moreover, I dislike the main change of this PR.

**Comment by vstinner** [comment]
@vadmium, @serhiy-storchaka, @berkerpeksag: I propose to reject this PR, what do you think?

**Comment by vstinner** [comment]
> How can I ask for a rerun of the CLA check plus add some labels?

I fixed the CLA label by removing the old red "CLA not signed" label: a new green "CLA signed" came automatically ;-)

**Comment by berkerpeksag** [comment]
I agree with Victor. Thank you for opening a PR to improve Python, bu we usually prefer to avoid doing style-only changes in the codebase.

**Comment by serhiy-storchaka** [comment]
I concur.

**Pull request closed without merging**